geoportailv3/static/js/infobar/elevationdirective.js

```diff
@@ -46,26 +47,33 @@ app.module.directive('appElevation', app.elevationDirective);
  * @ngInject
  * @constructor
  * @param {angular.$http} $http
+ * @param {app.GetElevation} appGetElevation
  * @param {ngeo.Debounce} ngeoDebounce
- * @param {string} elevationServiceUrl
  */
 app.ElevationDirectiveController =
-    function($http, ngeoDebounce, elevationServiceUrl) {
+    function($http, appGetElevation, ngeoDebounce) {
```

Comment:
Very minor but we usually place ngeo services before app services in the function arguments.